Source/DafnyLanguageServer.Test/Synchronization/DeclarationLocationMigrationTest.cs

```diff
@@ -82,7 +81,6 @@ class C {
       );
       var document = await Documents.GetResolvedDocumentAsync(documentItem.Uri);
       Assert.IsNotNull(document);
-      Assert.IsFalse(document.SymbolTable.Resolved);
```

Comment:
How do you justify removing this assertion? Don't we want to test that the document's symbol table was not resolved?

Reply:
Why would you want to test this? It seems like it's testing implementation instead of behavior, which makes the test brittle. For migrated symbol tables, .Resolved is always false but it's never read so the value doesn't matter.